UPGRADE.md

```diff
@@ -1,16 +1,7 @@
-## Upgrade steps from v. 4.2.x to v. 4.3 for native/hybrid apps
+## Upgrade steps from v. 4.3.x to v. 5.0 for native/hybrid/react native apps
 
-To upgrade native and hybrid, we strongly recommend creating a new app from the app templates in [the forceios npm package](https://npmjs.org/package/forceios), then migrating the artifacts specific to your app into the new template. The other recommended approach would be to upgrade using [Cocoapods](https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Specs). Read on if you prefer to update the Mobile SDK artifacts in your existing hybrid app.
+To upgrade native, hybrid and react native apps, we strongly recommend creating a new app from the app templates in [the forceios npm package](https://npmjs.org/package/forceios), then migrating the artifacts specific to your app into the new template. The other recommended approach would be to upgrade using [Cocoapods](https://github.com/forcedotcom/SalesforceMobileSDK-iOS-Specs).
 
-### Hybrid 4.2.x to 4.3 upgrade
+See the [Mobile SDK Development Guide](https://github.com/forcedotcom/SalesforceMobileSDK-Shared/blob/master/doc/mobile_sdk.pdf?raw=true) for more information about developing hybrid apps with the Mobile SDK and Cordova.
 
-Upgrading your hybrid app from 4.2.x to 4.3 should be a simple matter of upgrading the Salesforce Cordova plugins themselves. This can be done by using the Cordova command-line tool to remove, then re-add the plugin:
```

Comment:
Removed this section since too much has changed in `5.0`. It's best to regenerate to ensure a clean start.